test/unit/models/logs.test.js

```diff
@@ -0,0 +1,40 @@
+const chai = require("chai");
+const { expect } = chai;
+
+const cleanDb = require("../../utils/cleanDb");
+const logsQuery = require("../../../models/logs");
+const cacheData = require("../../fixtures/cloudflareCache/data");
+
+describe("Logs", function () {
+  after(async function () {
+    await cleanDb();
+  });
+
+  describe("Fetch purged cache logs", function () {
+    it("Should fetch the purged cache logs", async function () {
+      const { type, meta, body } = cacheData.cacheLogs[0];
+      const userId = meta.userId;
+
+      await logsQuery.addLog(type, meta, body);
+      const data = await logsQuery.fetchCacheLogs(userId);
+
+      expect(data[0].timestamp).to.be.an("object");
+      expect(data[0].timestamp._seconds).to.be.a("number");
```

Comment:
nit: is it possible to enforce that number is between 0-30

Reply:
Couldn't get you, how can we enforce the seconds to be in the range of 0-30 and what's the benefit of doing it?

Reply:
you are just checking that it's a number, it can be any number like 100 but seconds are only between 0-59 so.

The benefit would be better testing.

Also, it's an **NIT** comment.